apps/web/components/Charts/v2/utils/format.ts

```diff
@@ -27,12 +27,15 @@ export const formatDate = (
 export const formatValueForPresentation = (
   activeLocale: Locale,
   possiblyRawValue: number | string,
-  reduceAndRoundValue = true,
+  reduceAndRoundValue: boolean | null | undefined,
```

Comment:
I wonder if it'd be better if we keep it the way it is (defaults to true) and when calling this function we pass in the reduceAndRoundValue ?? true.